faq.yaml

```diff
@@ -54,33 +53,34 @@ en:
         msg.value greater than 0 for the entirety of the 3, 6 or 12-month lock durations”,
         and in these cases “that signal can be treated as a lock for the purposes of the allocation award”. 
 
-        By removing the ability to signal on behalf of a contract, Straightedge has solved for Generalized
-        Locks and other edge case vulnerabilities. However, it is still worth explicitly pointing out some
+        By removing the ability to signal on behalf of a contract, Straightedge has ntended to reduce vulnerabilities by
```

Comment:
Typo: "ntended"

Reply:
The point here was to say that, we that generalized locks are a moot point given that we "signalling on behalf of a contract" is removed.  However, we want to share some reasons why the generalized lock is especially harmful, even more so than a normal "contract signal".